java/light-4j/config.yaml

```diff
@@ -0,0 +1,14 @@
+framework:
+  website: doc.networknt.com
+  version: 1.6.23
```

Comment:
could you use `1.6` ?

Reply:
Is there any requirement for the version of JDK? The 1.6.x is for JDK 8 and 2.0.x is for JDK 11. Most users are using the JDK11 version which is a little bit faster.

Reply:
understood, I prefer to keep on jdk 8 to be as-fair-as possible in `java` **frameworks**. somes, at least, `act` is not compatible with `11` yet

Reply:
That is Cool. Let's keep the JDK8 then. Thanks for the input.